fili-core/src/test/groovy/com/yahoo/bard/webservice/table/availability/PermissiveAvailabilitySpec.groovy

```diff
@@ -0,0 +1,117 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.table.availability
+
+import com.yahoo.bard.webservice.data.config.names.TableName
+import com.yahoo.bard.webservice.metadata.TestDataSourceMetadataService
+import com.yahoo.bard.webservice.table.Column
+import com.yahoo.bard.webservice.table.resolver.DataSourceConstraint
+import com.yahoo.bard.webservice.util.SimplifiedIntervalList
+
+import org.joda.time.Interval
+
+import spock.lang.Specification
+import spock.lang.Unroll
+
+import java.util.stream.Collectors
+
+class PermissiveAvailabilitySpec extends Specification  {
+
+    PermissiveAvailability permissiveAvailability
+    TableName tableName
+    Column column1
+    Column column2
+    Interval interval1
+    Interval interval2
```

Comment:
Grouping the ones with the same type into a single line make things look cleaner